packages/functional-tests/pages/settings/totp.ts

```diff
@@ -67,7 +67,7 @@ export class TotpPage extends SettingsLayout {
     await this.submit();
     await this.setRecoveryCode(recoveryCodes[0]);
     await this.submit();
-    credentials.secret = secret;
+    credentials.secret = secret === null ? undefined : secret;
```

Comment:
I considered this, but that would set secret to undefined if secret was an empty string. Probably wouldn't happen, but I really just want to set it undefined when it's value was null.

Reply:
Instead of `||` wouldn't it be [`??`](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing)?

> _"The nullish coalescing (`??`) operator is a logical operator that returns its right-hand side operand when its left-hand side operand is `null` or `undefined`, and otherwise returns its left-hand side operand."_

```js
credentials.secret = secret ?? undefined;
```

So if `secret` is `null` or `undefined` we'd set it's value to `undefined` otherwise use the value of `secret`.

The PR is already merged and this is a bit moot, but&hellip; nullish is fun and I don't think we could use the [nullish coalescing assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_assignment) operator 🤷.